src/helpers/flare.ts

```diff
@@ -61,7 +61,22 @@ export const sendToDatadog = async (
       const errResponse: string = (err.response?.data.error as string) ?? ''
       const errorMessage = err.message ?? ''
 
-      throw Error(`Failed to send flare file to Datadog Support: ${errorMessage}. ${errResponse}\n`)
+      let message = `Failed to send flare file to Datadog Support: ${errorMessage}. ${errResponse}\n`
+      const code = err.response?.status
+      // The error message doesn't say why there was an error. All it says is:
+      // "[Error] Failed to send flare file to Datadog Support: Request failed with status code 500."
+      // Therefore, we need to add an explanation to clarify when the code is 500 or 403.
+      switch (code) {
```

Comment:
Also, the Flare endpoint provides an `errors` field, why don't we use it?

Reply:
We have a `formatBackendErrors()` function which uses this `errors` field:

https://github.com/DataDog/datadog-ci/blob/7ce750567e0f28506b88b3193ac176e30656a40a/src/commands/synthetics/api.ts#L40-L56

Feel free to move it and reuse if it suits your needs 😉

Reply:
> We have a `formatBackendErrors()` function which uses this `errors` field:
> 
> https://github.com/DataDog/datadog-ci/blob/7ce750567e0f28506b88b3193ac176e30656a40a/src/commands/synthetics/api.ts#L40-L56
> 
> Feel free to move it and reuse if it suits your needs 😉

Thanks for the suggestion! I don't think we'll need this here because we want some custom messages for handling errors